tests/ui/precondition-checks/cfg-ub-checks-no.rs

```diff
@@ -0,0 +1,19 @@
+//@ run-pass
+//@ compile-flags: -Cdebug-assertions=yes -Zub-checks=no
```

Comment:
The value of `-Cdebug-assertions` should not matter here, right? We only want to check `-Zub-checks`, if so can we omit it.

It would be more faithful, imo.

```suggestion
//@ compile-flags: -Zub-checks=no
```

Reply:
I don't have a strong opinion, so I'll make the change as you suggest. I applied this pattern in many cases to ensure that the flag is overriding the value of `-Cdebug-assertions`, because that is its only use case.